base/scripts/code.game.cs

```diff
@@ -727,6 +726,9 @@ function remoteSetArmor(%player, %armorType)
     else if (%killerId == %playerId)
     {
         //suicide
+        %playerCratered = false;
+        if(%damageType == $LandingDamageType) { %playerCratered = true; }
+        

```

Comment:
can remove this if you apply the below change
```suggestion
```

Reply:
no, because the game switches all damage types that are suicide into $suicidedamagetype that is why the check must be done before the switch occurs, we need to determine if it is fall damage.

Reply:
makes sense - ty